app/models/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event < ApplicationRecord
+  validates :description, :location, :date, presence: true
+  validates :description, length: { in: 10..120 }
+  validates :location, length: { in: 5..50 }
```

Comment:
Since you already validated the length of `description` and `location`, that validation automatically covers for `presence: true`, so you can remove `:description` and `:location` from `L2` to avoid repetition.